src/libraries/Common/tests/Tests/System/IO/PathInternal.Tests.cs

```diff
@@ -64,345 +64,5 @@ public void GetCommonPathLength(string first, string second, bool ignoreCase, in
         {
             Assert.Equal(expected, PathInternal.GetCommonPathLength(first, second, ignoreCase));
         }
-
```

Comment:
I'll bring back these unit tests, and simply update the method that verifies these expected/actual paths. That way we can be 100% sure the unit tests that were passing before, were not broken.